package-lock.json

```diff
@@ -594,1176 +428,479 @@
         "node": ">=10.0.0"
       }
     },
-    "node_modules/@esbuild/aix-ppc64": {
+    "node_modules/@esbuild/win32-x64": {
```

Comment:
@jingjiajie, the changes in this file look far beyond your change in the `package.json` file. Would you please install with existing lock file first, then only apply the corresponding change in `package.json` file?